### PR TITLE
Fix DeathTest failure on travis by not running DeathTests

### PR DESCRIPTION
**Pull request by crtrott**
Also fixing the naming issue of the tests.

### CODE REVIEW

**Comment by crtrott** [comment]
I had disabled the wrong OS ...

**Comment by dalg24** [comment]
Retest this please

**Comment by crtrott** [comment]
Fixed clang format.

**Comment by masterleinad** [comment]
Retest this please.

**Comment by crtrott** [comment]
had to update the exclude list in travis.yml :-(

**Comment by crtrott** [comment]
Woohoo Passed!

**Comment by dalg24** [comment]
@junghans please review

File: .travis.yml [hunk 1]
```diff
@@ -33,8 +33,8 @@ env:
   - 
   - BACKEND="OPENMP"
 #  - BACKEND="PTHREAD"
-  - CMAKE_BUILD_TYPE=Debug COVERAGE=yes
-  - CMAKE_BUILD_TYPE=Debug BACKEND="OPENMP" COVERAGE=yes
+  - CMAKE_BUILD_TYPE=Debug COVERAGE=yes GTEST_FILTER="-*DeathTest*"
+  - CMAKE_BUILD_TYPE=Debug BACKEND="OPENMP" COVERAGE=yes GTEST_FILTER="-*DeathTest*"
```
Comment (dalg24) [inline .travis.yml]: You are filtering out the DeathTest beyond the two builds that were failing.  Just making sure you realize that.

**Pull request merged**